### DISCUSSION TITLE
Governor limit proposal for Terra Classic after hard fork

### DISCUSSION THREAD

**Discussion by aadam-10** [wormhole-foundation/wormhole#2087] [Governance] [answered]
This is a discussion thread about modifying Wormhole governor limits on Terra Classic in the wake of the recent hard fork.

**Comment by ckeun** [comment, marked as answer]
Looking at Terra Classic activity since 11/15/2022, the largest individual outbound transfer was ~$70,000. Most transfers were much less. The max notional transferred in a 24 hour window from the past couple weeks was about $70,000.
![image002 (1)](https://user-images.githubusercontent.com/5993370/206268773-9eb86da8-5061-4e3c-88b1-60ca15bbf10b.png)

![image001 (1)](https://user-images.githubusercontent.com/5993370/206269176-8deacf0c-aa69-4c72-9eb6-7e9def93d715.png)

**Reply by claudijd** [reply]
Based on the data above, what do folks think about reducing the limits to $500k/$50k?

**Reply by aadam-10** [reply]
$500k/$50k is live as of [2.14.4](https://github.com/wormhole-foundation/wormhole/releases/tag/v2.14.4)

**Comment by SEJeff** [comment]
For a bit of context, the [terra-rebels](https://www.terrarebels.net/) organization is a community friendly fork of terra classic. They opted to do a [hard fork to re-enable IBC](https://github.com/terra-rebels/classic/releases/tag/v1.0.4) and convinced enough terra classic validators to use their software that it happened.

This caused any rpc / validator node not on the terra rebels release to stop incrementing blocks.

Additionally, it appears that the rebels org leads by far in developer manpower. The original [classic-core](https://github.com/terra-money/classic-core) repo managed by TFL is in fact backporting stuff from the terra rebels fork into their own version after the fact e.g.:
* [Merge rebels cosmos-sdk](https://github.com/terra-money/classic-core/pull/786)
* [Merge commit changes from rebel release v1.0.4](https://github.com/terra-money/classic-core/pull/800)

There wasn't a release made in classic-core [until the day after the hardfork](https://github.com/terra-money/classic-core/releases/tag/v0.5.23) happened. So any guardians should be using the new codebase at https://github.com/terra-money/classic-core to prevent the same thing from happening again in the future.

This issue also gives a good idea of the timeline: https://github.com/terra-money/classic-core/issues/801
